src/algorithms/blocked_gibbs_algorithm.h

```diff
@@ -5,6 +5,7 @@
 #include "conditional_algorithm.h"
 
 class BlockedGibbsAlgorithm : public ConditionalAlgorithm {
+ protected:
   //!
   bool update_hierarchy_params() override { return true; }  // TODO ?
```

Comment:
Does this algorithm need true or false here?

Reply:
I'm voting for false, the ones that need true are Neal3 and the Split-and-Merge